dataland-frontend/tests/e2e/specs/company-metadata/CompanyUploadTest.ts

```diff
@@ -0,0 +1,103 @@
+import { describeIf } from "../../support/TestUtility";
+import { fillCompanyUploadFields } from "../../utils/CompanyUpload";
+
+describeIf(
+  "As a user, I want to be able to create new companies",
+  {
+    executionEnvironments: ["development"],
```

Comment:
Wird das nur hier gebaucht?

Reply:
Ne auch in dem DataConsistency und potenziell auch dem User-Registration test.